_clubs/naak/naak-history-and-accomplishments.md

```diff
@@ -0,0 +1,52 @@
+* Obtained two grants from the Issaquah Kiwanis club in 2008 for habitat restoration. Received an Environmental Achievement award in 2009 and a tree that was planted across from the HOA office.
+* Monitored the health of Yellow Lake by gathering water samples annually for the testing of e-coli forms, petroleum, herbicides, heavy metals and the overall health of the lake. We are happy to report all tests to-date have indicated the lake is healthy with exception of e-coli forms likely due to pet waste.   We thank all pet owners who diligently dispose of their pet waste.
+* Implemented a successful Monofilament Fishing Line Recovery and Recycling program at Yellow Lake in 2013. This program was then presented to Senator Mark Mullet and several state legislative committees.  It has now been implemented at over 200 locations within our state, including 74 city and county parks and piers, 137 Washington Department of Fish and Wildlife (WDFW) sites and 29 state parks.
+* Installed and annually maintain Wood Duck nesting boxes at Gander Pond. Wood Ducks and Hooded Mergansers use these nesting boxes during their spring nesting season every year.
+* Acted in an advisory capacity to Klahanie HOA management and Board of Directors. This currently includes recommended projects, status reports, and budget requests.  NAAK's work continues to be included in the annual budget.
+
+### What's Ongoing?
+
+While significant progress has been made, the presence of noxious weeds continues to pose a threat to the health and diversity of Klahanie’s native ecosystems and wildlife habitats.  Our ongoing stewardship keeps noxious weeds from re-establishing and spreading throughout the natural areas.
+
+Wildlife habitats have been dramatically improved because Klahanie’s natural areas now offer an abundant variety of native plants for food, nesting and protection.  The surrounding forests, wetlands, ponds and Yellow Lake support more than 100 species of birds, including: herons, woodpeckers, raptors, owls and a large variety of songbirds and waterfowl. These sightings have been documented by the Eastside Audubon Society in their Bird Surveys and Christmas Bird Counts.
+
+In addition, deer, little brown bats, shrews, coyotes, Douglas squirrels, beavers, possums, the occasional bobcat or Black bear and several other mammals frequent these areas.
+
+With the support of the Klahanie HOA Board of Directors, staff, and volunteers, the beauty of our natural areas and the wildlife that these habitats support can be enjoyed both today and in the future.
+
+Join our NAAK team and/or help by participating in sponsored work parties.  We will enjoy working with you.

```

Comment:
This came up in another area of the site. If we encourage people to join an event/group, we need to provide easily accessible contact information on the same page.